### PR TITLE
Part one answer

### PR DESCRIPTION
**Pull request by clarbarr**

### CODE REVIEW

**Comment by Kayla-SA-W** [comment]
Please include you villians.sql file. This cannot get graded until its added.

**Comment by clarbarr** [comment]
Added!

Thank you

**Comment by clarbarr** [comment]
Sorry, I am doing something wrong.  Let me take a closer look and get back to you.

Thanks,

**Comment by clarbarr** [comment]
Hi Kayla,

I’ve added the file.

Thank you,

Clarissa

File: villains.sql [hunk 1]
```diff
@@ -0,0 +1,32 @@
+CREATE DATABASE villains;
+CREATE USER 'villains'@'localhost' IDENTIFIED WITH mysql_native_password BY '$ThisisaTest3$';
+GRANT ALL ON villains.* TO 'villains'@'localhost';
+USE disney;
```
Comment (Kayla-SA-W) [inline villains.sql:4]: _Nit_ wrong database

**Comment by Kayla-SA-W** [comment]
-7 wrong database in .sql file prevents code from running properly
-5 poor commits 

Grade: 88

**Pull request closed without merging**